### ISSUE TITLE
Output: definitions.h:8:13: error: use of undeclared identifier 'TASK_COMM_LEN'\n

### ISSUE THREAD

**Issue by vt-alt** [iovisor/bpftrace#2246]
### What reproduces the bug?

When building bpftrace 0.15.0 for ALT Linux `tests/runtime-tests.sh` produces error:
```
[----------] 11 tests from regression
[ RUN      ] regression.access ctx struct field twice
[  FAILED  ] regression.access ctx struct field twice
        Command: /usr/src/RPM/BUILD/bpftrace-0.15.0/x86_64-alt-linux/src//bpftrace -e 'tracepoint:sched:sched_wakeup { if (args->comm == "asdf") { print(args->comm) } exit(); }'
        Unclean exit code: 1
        Output: definitions.h:8:13: error: use of undeclared identifier 'TASK_COMM_LEN'\n
[ RUN      ] regression.async_id_invalid_probe_expansion
[       OK ] regression.async_id_invalid_probe_expansion
[ RUN      ] regression.codegen_struct_save_nested
[       OK ] regression.codegen_struct_save_nested
[ RUN      ] regression.modulo_operator
[       OK ] regression.modulo_operator
[ RUN      ] regression.single_arg_wildcard_listing
[       OK ] regression.single_arg_wildcard_listing
[ RUN      ] regression.single_arg_wildcard_listing_tracepoint
[       OK ] regression.single_arg_wildcard_listing_tracepoint
[ RUN      ] regression.string compare with empty
[       OK ] regression.string compare with empty
[ RUN      ] regression.string compare with prefix
[       OK ] regression.string compare with prefix
[ RUN      ] regression.strncmp with prefix
[       OK ] regression.strncmp with prefix
[ RUN      ] regression.strncmp_checks_for_nul
[       OK ] regression.strncmp_checks_for_nul
[ RUN      ] regression.strncmp_n_longer_than_buffer
[       OK ] regression.strncmp_n_longer_than_buffer
[----------] 11 tests from regression
```

### bpftrace --info

```
====================
bpftrace --info:
====================
System
  OS: Linux 5.17.11-un-def-alt1 #1 SMP PREEMPT Wed May 25 18:26:15 UTC 2022
  Arch: x86_64

Build
  version: v0.15.0
  LLVM: 13.0.1
  ORC: v2
  foreach_sym: yes
  unsafe uprobe: no
  bfd: yes
  bpf_attach_kfunc: yes
  bcc_usdt_addsem: yes
  bcc bpf_attach_uprobe refcount: yes
  bcc library path resolution: yes
  libbpf: yes
  libbpf btf dump: yes
  libbpf btf dump type decl: yes
  libdw (DWARF support): no

Kernel helpers
  probe_read: yes
  probe_read_str: yes
  probe_read_user: yes
  probe_read_user_str: yes
  probe_read_kernel: yes
  probe_read_kernel_str: yes
  get_current_cgroup_id: yes
  send_signal: yes
  override_return: yes
  get_boot_ns: yes
  dpath: yes

Kernel features
  Instruction limit: 1000000
  Loop support: yes
  btf (depends on Build:libbpf): yes
  map batch (depends on Build:libbpf): yes
  uprobe refcount (depends on Build:bcc bpf_attach_uprobe refcount): yes

Map types
  hash: yes
  percpu hash: yes
  array: yes
  percpu array: yes
  stack_trace: yes
  perf_event_array: yes

Probe types
  kprobe: yes
  tracepoint: yes
  perf_event: yes
  kfunc: yes
  iter:task: yes
  iter:task_file: yes
```

**Comment by viktormalik** [comment]
Should be fixed by #2236.

**Comment by viktormalik** [comment]
Closing as fixed. Feel free to re-open if the issue persists.
